Packs/CortexAttackSurfaceManagement/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Cortex Attack Surface Management",
     "description": "Content for working with Attack Surface Management (ASM).",
     "support": "xsoar",
-    "currentVersion": "1.7.5",
+    "currentVersion": "1.7.6",
```

Comment:
Please check other PRs for version conflict.

Reply:
Updated to 1.7.9 and fixed branch that its merged into.